library/Vanilla/EmbeddedContent/Factories/PanoptoEmbedFactory.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace Vanilla\EmbeddedContent\Factories;
+
+use Vanilla\EmbeddedContent\AbstractEmbed;
+use Vanilla\EmbeddedContent\AbstractEmbedFactory;
+use Vanilla\EmbeddedContent\Embeds\PanoptoEmbed;
+
+/**
+ * PanoptoEmbedFactory class.
+ */
+class PanoptoEmbedFactory extends AbstractEmbedFactory {
+
+    /** @var int DEFAULT_HEIGHT */
+    private const DEFAULT_HEIGHT = 360;
+
+    /** @var int DEFAULT_WIDTH */
+    private const DEFAULT_WIDTH = 640;
```

Comment:
I am not sure if the embed URL returns frame ratio.
But in our other embed factory implementations we have it set up that way:
```
            'height' => $response['height'] ?? null,
            'width' => $response['width'] ?? null,